src/beautifiers/terraformfmt.coffee

```diff
@@ -0,0 +1,24 @@
+###
+Requires terraform installed
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Terraformfmt extends Beautifier
+  name: "terraformfmt"
+  link: "https://www.terraform.io/docs/commands/fmt.html"
+  isPreInstalled: false
+
+  options: {
+    Terraform: true
```

Comment:
Should be `false` indicating Language is supported *with no/zero support for options*.